crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -2,25 +2,33 @@
 //! This module contains functions that are used to verify blocks and transactions, and doesn't
 //! assume anything about the chainstate, so it can be used in any context.
 //! We use this to avoid code reuse among the different implementations of the chainstate.
-
+extern crate std;
```

Comment:
Use `floresta_common::prelude`. This breaks the `no-std` build for `floresta-chain`. You may need to add a `no-std` replacement to `SystemTime` though.

Reply:
Actually i did not write this line of code. was already there, just inserted a line... My rust-analyzer was bothering me about this so i disabled.